modules/arm_plugin/src/arm_converter/arm_converter_one_hot.cpp

```diff
@@ -2,88 +2,44 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include <details/ie_exception.hpp>
-
 #include "arm_converter/arm_converter.hpp"
 #include <ngraph/runtime/reference/one_hot.hpp>
 
 namespace ArmPlugin {
-template <typename Indices, typename OutputType>
-void wrap_one_hot(const Indices* arg,
```

Comment:
Why did you delete wrapper?

Reply:
I thought we don't need a function that does nothing but calls another function, so I fed ngraph::runtime::reference::xxx directly to MakeConversion(), like it is done [here](https://github.com/openvinotoolkit/openvino_contrib/blob/de5601166c0a5820fbf6f4644e7e3f74d921dcf8/modules/arm_plugin/src/arm_converter/arm_converter_cumsum.cpp#L18) or [here](https://github.com/openvinotoolkit/openvino_contrib/blob/de5601166c0a5820fbf6f4644e7e3f74d921dcf8/modules/arm_plugin/src/arm_converter/arm_converter_conv_backprop_data.cpp#L32) for example. But your question makes me think that I'm not getting it right. 
So could you please explain, why do we need the wrapper? To perform checks, like you mentioned in [this comment](https://github.com/openvinotoolkit/openvino_contrib/pull/52#discussion_r581815735)?

Reply:
We use wrappers to work with inconstant nodes. Instead of extraction a depth value at the graph construction stage, we pass an input node. The value will be extracted at the infer stage. We also need wrappers if `ngraph:: reference` has `char*` input value.

Reply:
Thank you! Will bring the wrapper back.